crates/nu-glob/src/lib.rs

```diff
@@ -966,7 +970,7 @@ mod test {
                 .and_then(|p| match p.components().next().unwrap() {
                     Component::Prefix(prefix_component) => {
                         let path = Path::new(prefix_component.as_os_str()).join("*");
-                        Some(path)
+                        Some(path.to_path_buf())
```

Comment:
Nit: why the `to_path_buf()` when it wasn't needed before?

Reply:
Good question, if you check https://github.com/rust-lang-nursery/glob/pull/105/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L1101 it was there but here looks like @jntrnr removed or at the time it was copied the `to_path_buf()` wasn't present